underlay/src/main/java/bio/terra/tanagra/query/bigquery/BQExecutor.java

```diff
@@ -170,6 +174,34 @@ public Pair<String, String> export(
     return Pair.of(getCloudStorageService().createSignedUrl(gcsUrl), fileName);
   }
 
+  public static String replaceFunctionsThatPreventCaching(
```

Comment:
Maybe worth adding this list of supported functions to the docs?

Reply:
Done.